ledger/src/bin/ledger.rs

```diff
@@ -6,7 +6,7 @@ fn main() {
 
         let now = redux::Instant::now();
 
-        let mut db = Database::<V2>::create(20);
+        let mut db = Database::<V2>::create(20, false);
```

Comment:
since most of the calls remain the same, wouldn't it be better to leave `create` as is and add a new `create_with_token_accounts` constructor for when it needs to be enabled?

Reply:
Yes I agree, that seems a good solution